tap_csv/client.py

```diff
@@ -134,18 +134,22 @@ def schema(self) -> dict:
                 break
             break
 
+        for column in header:
+            # Set all types to string
+            # TODO: Try to be smarter about inferring types.
+            properties.append(th.Property(column, th.StringType()))
+
         # If enabled, add file's metadata to output
         if self.config.get("add_metadata_columns", False):
             header = [
                 SDC_SOURCE_FILE_COLUMN,
                 SDC_SOURCE_FILE_MTIME_COLUMN,
                 SDC_SOURCE_LINENO_COLUMN,
             ] + header
-
-        for column in header:
-            # Set all types to string
-            # TODO: Try to be smarter about inferring types.
-            properties.append(th.Property(column, th.StringType()))
+            
```

Comment:
```suggestion

            properties.append(th.Property(SDC_SOURCE_FILE_COLUMN, th.StringType))
            properties.append(
                th.Property(SDC_SOURCE_FILE_MTIME_COLUMN, th.DateTimeType)
            )
```

@TyShkan linting was complaining about the formatting of this section. I think it this should fix it or you can run `poetry run tox -e format`.